packages/react-components/src/components/shared-components/StatusIcon/StatusIcon.tsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
-import './StatusIcon.css';
+import styled from 'styled-components';
 import type { StatusIconType } from '@iot-app-kit/core';
 import { STATUS_ICON_TYPE } from '@iot-app-kit/core';
 import { getIcons } from '../../../common/iconUtils';
 
+const Icon = styled.div`
+  position: relative;
+  margin-right: 3px;
+  top: 2px;
```

Comment:
should we setup a variables object to keep some standardized spacing? or we can use cloudscape and just use their variables in JS?

Reply:
Yes we should revisit this styling - I was looking to tackle that in a follow up and this PR focus on just porting over to styled components to allow it to be imported with the existing build system that doesn't handle `css` files correctly

Reply:
cool works for me!